service/src/main/java/com/commercetools/pspadapter/payone/notification/common/AppointedNotificationProcessor.java

```diff
@@ -33,8 +34,9 @@
      *
      * @param serviceFactory the services factory for commercetools platform API
      */
-    public AppointedNotificationProcessor(TenantFactory serviceFactory, TenantConfig tenantConfig) {
-        super(serviceFactory, tenantConfig);
+    public AppointedNotificationProcessor(TenantFactory serviceFactory, TenantConfig tenantConfig,
```

Comment:
I would call the first parameter tenantFactory instead of serviceFactory

Reply:
8da39a27